fastly/client.go

```diff
@@ -153,6 +167,17 @@ func (c *Client) init() (*Client, error) {
 	return c, nil
 }
 
+// Remaining returns the number of non-read requests left before
+// rate limiting causes a 429 Too Many Requests error.
+func (c *Client) Remaining() int {
+	return c.remaining
+}
+
+// Reset returns the next time the rate limiter's Remaining counter will be refilled.
+func (c *Client) Reset() time.Time {
```

Comment:
Maybe we rename to `RateLimitRemaining` and `RateLimitReset`.

Reply:
Done.